src/results.R

```diff
@@ -37,32 +57,17 @@ calculateRisk <- function(input, county_data) {
     sympt_covid_logodds <- (-1.32) - (0.01*age) + (0.44*sex_symp_val) + (1.75*"is_loss_smell_taste" %in% input$symptoms) + 
       (0.31*"is_cough" %in% input$symptoms) + (0.49*"is_fatigue" %in% input$symptoms) + (0.39*"is_skip_meal" %in% input$symptoms)
     
-    sympt_covid_risk <- logodds2risk(sympt_covid_logodds)
-    sympt_odds <- risk2odds(sympt_covid_risk)
-    exposure_risk <- odds2risk(risk2odds(community_exposure_risk)*sympt_odds)
+    sympt_covid_risk <- logodds2risk(sympt_covid_logodds) # option 1
```

Comment:
@pendlrobinson Do you mind expanding the comments "option 1", what does it mean?

Reply:
@xindyhu This is option #1 I mentioned in the Jira ticket where sympt_covid_odds = odds covid given symptoms. 
I will remove the comments when I am ready to complete the pull request.